package.json

```diff
@@ -61,15 +61,15 @@
     "watchify": "^2.4.0"
   },
   "scripts": {
-    "frontend:build": "./node_modules/browserify/bin/cmd.js -t partialify -d -p [minifyify --map mypads.map.json --output static/js/mypads.map.json] -o static/js/mypads.js frontend.js",
-    "frontend:watch": "./node_modules/watchify/bin/cmd.js -t partialify -o static/js/mypads.js frontend.js",
-    "functest:build": "./node_modules/browserify/bin/cmd.js -o spec/frontend/mypads.js spec/frontend/entry.js",
-    "functest:watch": "./node_modules/watchify/bin/cmd.js -o spec/frontend/mypads.js spec/frontend/entry.js",
+    "frontend:build": "node_modules\\.bin\\browserify -t partialify -d -p [minifyify --map mypads.map.json --output static\\js\\mypads.map.json] -o static\\js\\mypads.js frontend.js",
```

Comment:
Should not commit this, please revert also..  Note that this and package.lock.json should be retained as your local copy, but should not be git added to your commit